### ISSUE TITLE
cron example and code need to be refactored to help testing it in a delay shorter than 1min

### ISSUE THREAD

**Issue by ccoVeille** [gofr-dev/gofr#916]
We may have to use ways to speed up the timer

_Originally posted by @ccoVeille in https://github.com/gofr-dev/gofr/pull/831#pullrequestreview-2223167031_

**Comment by ccoVeille** [comment]
I like what they did for the tests in this package

https://github.com/mileusna/crontab

**Comment by srijan-27** [comment]
I am thinking of adding optional `seconds` part in cron format as the first part.
It will not be a breaking change as we will keep this as optional, and will continue to process the existing format.
@ccoVeille - what are your thoughts on this?

**Comment by ccoVeille** [comment]
I'm afraid, I don't understand what you mean

Could you rephrase or provide an example?

Thanks

**Comment by srijan-27** [comment]
Currently, supported cron format is: `* * * * *` i.e (5 part) format denoting minute, hour, day, month and day of week respectively.
Proposed enhancement: Change cron format to `* * * * * *` i.e (6 part) format with first part being seconds which will be optional, and rest would be the same as above.
